docs/src/operate/configuration/config-mapper-mqtt-topics.md

```diff
@@ -0,0 +1,64 @@
+# How to configure MQTT topics to subscribe for mappers
+
+The cloud-specific mappers subscribe to the reserved MQTT topics and convert incoming MQTT messages to cloud-specific messages.
+In an advanced use case, such as using more than one cloud mappers for the same device,
+you may want to customize the MQTT topics that each cloud mapper subscribes to.
```

Comment:
Though doesn't this setting more control which tedge topics are subscribe to by each mapper, and not the data coming from each cloud?

Reply:
True. For example, `c8y/s/us` is not a candidate to be configured by tedge-mapper-c8y.